### PR TITLE
Update release tag of ECSchemaXmlContext class

### PR DESCRIPTION
**Pull request by akshay05n**

### CODE REVIEW

File: core/backend/src/ECSchemaXmlContext.ts [hunk 1]
```diff
@@ -30,18 +32,34 @@ export class ECSchemaXmlContext {
     return this._nativeContext;
   }
```
Comment (grigasp) [inline core/backend/src/ECSchemaXmlContext.ts:33]: Just making sure you really want `nativeContext` to be `@public`.

Comment (pmconne) [reply core/backend/src/ECSchemaXmlContext.ts:33]: Definitely not, it needs to remain `internal`.

Comment (pmconne) [reply core/backend/src/ECSchemaXmlContext.ts:33]: Is there any reason for the public accessor to exist? Shouldn't the TypeScript `ECSchemaXmlContext` APi provide all the functionality any consumers could need?

Comment (akshay05n) [reply core/backend/src/ECSchemaXmlContext.ts:33]: The getter method is required by the IModelDb class and SchemaUtils.ts file, so it needs a public accessor. However, I will change the release tag to `internal` because we do not want to expose something from the native package.

File: core/backend/src/ECSchemaXmlContext.ts [hunk 2]
```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
 /** @internal */
 export type SchemaMatchType = IModelJsNative.ECSchemaXmlContext.SchemaMatchType;
 
-/** @internal */
+/** Context for deserializing ECSchemas
```
Comment (pmconne) [inline core/backend/src/ECSchemaXmlContext.ts:20]: Can you be more specific about what it's for, how it's used? e.g., link to other APIs that require one of these contexts?

File: core/backend/src/ECSchemaXmlContext.ts [hunk 3]
```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
...
   }
 
+  /** @internal */
   public get nativeContext(): IModelJsNative.ECSchemaXmlContext {
     assert(undefined !== this._nativeContext);
     return this._nativeContext;
   }
 
+  /**
+   * Adds a file path that should be used to search for a matching schema name
```
Comment (pmconne) [inline core/backend/src/ECSchemaXmlContext.ts:37]: Please specify in the docs the order in which paths are searched if I call addSchemaPath multiple times. Same for schema locators.

File: core/backend/src/ECSchemaXmlContext.ts [hunk 4]
```diff
@@ -17,31 +17,52 @@ export type SchemaKey = IModelJsNative.ECSchemaXmlContext.SchemaKey;
...
+   * Set the last locater to be used when trying to find a schema
+   * @param locater Locater that should be used as the last locater when trying to find a schema
+   * @internal
+   */
   public setSchemaLocater(locater: IModelJsNative.ECSchemaXmlContext.SchemaLocaterCallback): void {
     this.nativeContext.setSchemaLocater(locater);
   }
 
+  /**
+   * Adds a schema locater as first to the current context
```
Comment (pmconne) [inline core/backend/src/ECSchemaXmlContext.ts:54]: I assume "first" means "searched for schema matches before other locaters". Can you rephrase the docs so I don't have to make assumptions?

File: core/backend/src/ECSchemaXmlContext.ts (hunk 2 above)
Comment (pmconne) [reply core/backend/src/ECSchemaXmlContext.ts]: I did some cleanup on your docs. Your code example used an `internal` API (one of many reasons to put code examples into example-code-snippets package). Resolve this comment if you're okay with my changes.

**Review by pmconne — APPROVED** [review]

**Review by pmconne — CHANGES_REQUESTED** [review]
Why is this targeting the `imodel02` branch instead of `master`?

**Comment by pmconne** [comment]
> I created this branch from imodel02 since https://github.com/iTwin/itwinjs-core/pull/5574 started causing PR validation failures.

No validation is happening for your PR because of its target branch and lack of corresponding native changes.
Since all you are changing is documentation and release tags, I think you should target master.

**Review by pmconne — APPROVED** [review]
